Services/StopPointManager.php

```diff
@@ -104,6 +115,7 @@ public function getStopPoint($externalStopPointId, $timetable, $externalCoverage
             $this->stopPoint->setTimetable($timetable);
         }
         $this->initTitle($externalCoverageId);
+        $this->initCity($externalCoverageId);
         $this->initStopPointCode($externalCoverageId);
```

Comment:
$this->initTitle($externalCoverageId); and  $this->initStopPointCode($externalCoverageId); do the same request /v1/coverage/jdr/stop_points/stop_point:TAD:SP:92002/?depth=1&show_codes=true

Is there a way to do the request once?